### ISSUE TITLE
Issue with AWS implementation

### ISSUE THREAD

**Issue by Qiangong2** [ConnorMWGraham/Minecraft-Nintendo-Switch-Private-Server#1]
I posted this on reddit, but thought it may fit better here.

I built a nukkitx server on an aws instance, and it works great. However, when I tried to use this so that my switch could connect, it did not work.

I replaced the ip with the server ip, and got past docker-compose up. However, when I went to put in the server ip in the primary dns, it did not connect. Just sent me to the lifeboat server.

**Comment by ConnorGraham** [comment]
Can you confirm that in your nintendo switch network settings you changed the primary dns server to the server hosting your DNS? 
![image](https://user-images.githubusercontent.com/55450371/72021462-0ca83c00-323c-11ea-87e8-2c6048acd843.png)

**Comment by ConnorGraham** [comment]
Could be related to new hostname 
https://github.com/ConnorGraham/Minecraft-Nintendo-Switch-Private-Server/issues/2

**Comment by Qiangong2** [comment]
Yeah, the DNS is pointed to my aws instance. Everything is functional up to that point

**Comment by Qiangong2** [comment]
@ConnorGraham Did you look into this at all?
